ios/RNSentry.m

```diff
@@ -85,18 +85,28 @@ + (BOOL)requiresMainQueueSetup {
         }
     }
 
+
     // Enable the App start and Frames tracking measurements
     if ([mutableOptions valueForKey:@"enableAutoPerformanceTracking"] != nil) {
         BOOL enableAutoPerformanceTracking = (BOOL)[mutableOptions valueForKey:@"enableAutoPerformanceTracking"];
 
         PrivateSentrySDKOnly.appStartMeasurementHybridSDKMode = enableAutoPerformanceTracking;
+#if TARGET_OS_IPHONE || TARGET_OS_MACCATALYST
         PrivateSentrySDKOnly.framesTrackingMeasurementHybridSDKMode = enableAutoPerformanceTracking;
+#endif
     }
 
+
     [SentrySDK startWithOptionsObject:sentryOptions];
 
+#if TARGET_OS_IPHONE || TARGET_OS_MACCATALYST
+    BOOL appIsActive = [[UIApplication sharedApplication] applicationState] == UIApplicationStateActive;
+#else
+    BOOL appIsActive = [[NSApplication sharedApplication] isActive];
```

Comment:
I'm not entirely sure this is the equivalent of `UIApplicationStateActive` for `NSApplication`.
`UIApplication` has 3 states, while `isActive` returns a boolean.
on Flutter, we register a listener, https://github.com/getsentry/sentry-dart/blob/main/flutter/ios/Classes/SentryFlutterPluginApple.swift#L35-L54=
and flip a flag if its active `didReceiveDidBecomeActiveNotification`.
I don't recall the reason, maybe this is because `NSApplication` does not have the equivalent of `applicationState` for `UIApplication`

Reply:
@philipphofmann or @brustolin ideas?

Reply:
I think the application state on iOS has to do more with which app is on the forefront, whereas on macOS apps can still be active but not key. In either case this is only the app start notification (from what I can see) so it shouldn't really change too much as long as the notification that SDK has been initialized is dispatched.

Reply:
Using `isActive` should be fine here, as we only post it when initializing the SDK. We use the `SentryHybridSdkDidBecomeActive`  for OOM, which doesn't work for macOS and for sessions.